### ISSUE TITLE
[3.1.3]  test failure with scipy 1.10

### ISSUE THREAD

**Issue by olebole** [dfm/emcee#453]
**General information:**

This is a forward of [Debian#1029688](https://bugs.debian.org/1029688).

- emcee version: 3.1.3
- platform: Debian unstable/experimental
- installation method (pip/conda/source/other?): Debian package build

**Problem description:**

When running the tests on the Debian package with scipy upgraded to 1.10.0, we observe test errors. The errors are IndexErrors.

### Expected behavior:

All tests should pass

### Actual behavior:

A sample from the failing test log is

```
_______________________________ test_uniform_de ________________________________

kwargs = {}

    def test_uniform_de(**kwargs):
>       _test_uniform(moves.DEMove(), **kwargs)

/usr/lib/python3/dist-packages/emcee/tests/integration/test_de.py:19: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/dist-packages/emcee/tests/integration/test_proposal.py:101: in _test_uniform
    ks, _ = stats.kstest(samps[::100], "uniform")
/usr/lib/python3/dist-packages/scipy/_lib/_util.py:700: in wrapper
    return fun(*args, **kwargs)
/usr/lib/python3/dist-packages/scipy/stats/_stats_py.py:8383: in kstest
    return ks_1samp(xvals, cdf, args=args, alternative=alternative,
/usr/lib/python3/dist-packages/scipy/_lib/_util.py:700: in wrapper
    return fun(*args, **kwargs)
/usr/lib/python3/dist-packages/scipy/stats/_stats_py.py:7759: in ks_1samp
    Dplus, dplus_location = _compute_dplus(cdfvals, x)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cdfvals = array([[0.3753341 ],
       [0.        ],
       [0.        ],
       [0.        ],
       [0.        ],
       [0.168...[0.89123376],
       [1.        ],
       [0.55866734],
       [0.74500648],
       [0.        ],
       [0.        ]])
x = array([[ 0.3753341 ],
       [-1.0110354 ],
       [-0.56077189],
       [-0.90308162],
       [-0.99837366],
       [...123376],
       [ 2.13529302],
       [ 0.55866734],
       [ 0.74500648],
       [-0.65506003],
       [-0.83396273]])

    def _compute_dplus(cdfvals, x):
        """…"""
        n = len(cdfvals)
        dplus = (np.arange(1.0, n + 1) / n - cdfvals)
        amax = dplus.argmax()
>       loc_max = x[amax]
E       IndexError: index 1279 is out of bounds for axis 0 with size 640

/usr/lib/python3/dist-packages/scipy/stats/_stats_py.py:7586: IndexError
```

### What have you tried so far?:

--

### Minimal example:

```shell
pytest-3 --pyargs emcee
```

**Comment by dfm** [comment]
Thanks! Fixed in #454. I'll draft a new release ASAP.

**Comment by dfm** [comment]
Done: https://pypi.org/project/emcee/3.1.4/

Let me know if there are further issues.
